### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Moved descriptions and content to the documentation for consistency.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,7 @@
 [![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-incident-response/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-incident-response)
 [![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-incident-response)](https://github.com/mattermost/mattermost-plugin-incident-response/releases/latest)
 
-Incident Response allows your team to coordinate, manage, and resolve incidents from within Mattermost.
-
-![Incident Response plugin screenshot](assets/center_channel.png)
```
Comment (agarciamontoro) [inline README.md:9]: Should we keep the screenshot (this one or a different one)? I always like when a repo has an image showcasing what the product is.

**Review by agarciamontoro — COMMENTED** [review]
Fine with this! I only left a comment regarding the screenshot.

File: README.md (hunk 1 above)
Comment (justinegeffen) [reply README.md:9]: I've kept it and put it on the front page of the documentation. The goal is to have all the repos looking like this one in terms of the README content and how it links to the docs: https://github.com/mattermost/mattermost-plugin-jira. 

I do agree with your point about the screenshot in the repo landing page. It'll be easy to standardize across the other repos and will look better. :)

**Review by agarciamontoro — APPROVED** [review]
Thank you, @justinegeffen :tada:

**Pull request merged**